app/styles/_shame.scss

```diff
@@ -8,13 +8,22 @@
   }
 }
 
+.au-c-toolbar__group--row.comments-row {
+  > ul {
+    margin-left: 0;
+    > .au-o-grid__item {
+      padding-left: 0;
+    }
+  }
+}
+
 .au-c-toolbar_space-between{
-  display: flex;
-  justify-content: space-between;
+  @extend .au-u-flex;
+  @extend .au-u-flex--between;
 }
 
 .au-word-wrap {
-  word-wrap: break-word;
+  @extend .au-u-word-break;
```

Comment:
Nitpick: These changes seem unneeded? I think a better change would be to use the utils directly where needed.